### PR TITLE
migrating to v1

### PR DESCRIPTION
**Pull request by mesudpasic**

### CODE REVIEW

File: Module.yaml [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
+categories:                             # Array of category that will be used to group the module in the UI.
+  - 'Scheduler'
+type: 'Processing'      # Module type (Input, Processing, Output) that will be used to group the module in the UI.
 image:
-  name: weevenetwork/scheduler
-  tag: latest
-environments:
-  - key:  DB_REST_API
+  name: 'weevenetwork/scheduler'     # Image name from the docker registry.
+  tag: 'latest'                               # Image tag from the docker registry.
```
Comment (nithinsaii) [inline Module.yaml]: set `tag` to `v1.0.0`

**Review by nithinsaii — CHANGES_REQUESTED** [review]

File: Module.yaml (hunk 1 above)
Comment (mesudpasic) [reply Module.yaml]: done.

File: makefile [hunk 2]
```diff
@@ -1,25 +1,49 @@
-SHELL := /bin/bash
+SHELL := /bin/bash # to enable source command in run_app
+
 MODULE=weevenetwork/scheduler
+VERSION_NAME=v1.0.0
+
+lint:
+	npm run format
+.phony: lint
```
Comment (nithinsaii) [inline makefile]:
```suggestion
lint:
	npm run lint
.phony: lint

lint-fix:
	npm run lint-fix
.phony: lint-fix

format:
	npm run format
.phony: format
```

File: docker/docker-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    : "${INGRESS_HOST:?Need to set INGRESS_HOST environment variable to string}"
+    : "${INGRESS_PORT:?Need to set INGRESS_PORT environment variable to string}"
+else
+    echo "Unrecognized MODULE_TYPE = $MODULE_TYPE, choose from Input, Processing, Output"
+    exit 1
+fi
+echo "[ENTRYPOINT] Environment validated."
+
+# CALL THE MAIN SCRIPT
+npm run start $@
```
Comment (nithinsaii) [inline docker/docker-entrypoint.sh]:
```suggestion
npm run start "$@"
```

File: makefile (hunk 2 above)
Comment (mesudpasic) [reply makefile]: done.

File: docker/docker-entrypoint.sh (hunk 3 above)
Comment (mesudpasic) [reply docker/docker-entrypoint.sh]: done

**Review by nithinsaii — APPROVED** [review]
Module.yaml looks good and container runs fine locally.

**Pull request merged**